cdap-data-fabric/src/test/java/co/cask/cdap/data2/increment/hbase/AbstractIncrementHandlerTest.java

```diff
@@ -73,10 +79,10 @@ public static void tearDown() throws Exception {
 
   @Test
   public void testIncrements() throws Exception {
-    String tableName = "incrementTest";
-    createTable("incrementTest");
+    TableId tableId = TableId.from(Constants.DEFAULT_NAMESPACE, "incrementTest");
```

Comment:
failing to understand why this class had to change. can you link me to the failure?